test/testcafe/spec/ChallengeOktaVerifyFastPassView_spec.js

```diff
@@ -47,7 +48,7 @@ const loopbackFallbackMock = RequestMock()
   .respond(identifyWithUserVerificationCustomURI);
 
 const identifyWithLaunchAuthenticatorHttpCustomUri = JSON.parse(JSON.stringify(identifyWithUserVerificationCustomURI));
-const mockHttpCustomUri = 'http://localhost:3000/launch-okta-verify';
+const mockHttpCustomUri = 'http://localhost:6512/launch-okta-verify';
```

Comment:
This change seems unrelated to profile enrollment and I didn't see anything in the description for this. Is this intentional ?

Reply:
Yeah, due to change in `webpack.playground`, the playground would support any URL.
Before the change, `http://localhost:3000/launch-okta-verify` will respond 404 since the test case didn't care whether the URL works or not. But now the URL loads correctly and will generates additional logs to console hence test failed.